README.md

```diff
@@ -356,6 +402,8 @@ You can then run the following two commands in order to be able to connect to Gr
 export POD_NAME=$(kubectl get pods --namespace grafana -l "app=grafana" -o jsonpath="{.items[0].metadata.name}")
 kubectl --namespace grafana port-forward $POD_NAME 3000
 ```
+SR: This is impossible, 2 sections up we already port-forwarded 3000 to our application, that ports in use! I used "4000:3000" here and in the URL below
```

Comment:
not sure what you want to do here

Reply:
Swap to another port - I think this means we need to open a new terminal window because of port-forward?

Reply:
Yes. All the port forwards are like that, there were 3 in a row.... I had to google around to assure myself they weren't buggy, and it was intended they not return.